x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/search_experience.tsx

```diff
@@ -57,6 +57,8 @@ export const SearchExperience: React.FC = () => {
 
   // TODO const sortFieldsOptions = _flatten(fields.sortFields.map(fieldNameToSortOptions)) // we need to flatten this array since fieldNameToSortOptions returns an array of two sorting options
   const sortingOptions = [...DEFAULT_SORT_OPTIONS /* TODO ...sortFieldsOptions*/];
+  const openCustomizationModal = () => setShowCustomizationModal(true);
+  const closeCustomizationModal = () => setShowCustomizationModal(false);
```

Comment:
super duper minor optional nit (feel free to ignore for time reasons), it would be nice to move this to after line 49 to keep customizationModal related logic together

Reply:
Good point. d60da57bc20f6c5210e68e942caa3f7d3f7389b2